modules/claims_api/app/controllers/claims_api/base_form_controller.rb

```diff
@@ -47,7 +47,28 @@ def auth_headers
 
     def documents
       document_keys = params.keys.select { |key| key.include? 'attachment' }
-      params.slice(*document_keys).values
+      params.slice(*document_keys).values.map do |document|
+        if document.is_a?(String)
+          decode_document(document)
+        else
+          document
+        end
+      end.compact
+    end
+
+    def decode_document(document)
+      match = document.match(/\;base64\,(.+?)$/)
```

Comment:
Should we be doing a match here? this could be very slow on large files. If we dont want to do a start with, maybe we could take the first x chars and do the match on that vs the whole document

Reply:
Good point, You will not see any commas, colons, or double quotes in a Base64 encoded string so maybe we do a document.split(',').last and it should give us the same base64 string we need.

Reply:
i like the _taking the first x chars to check if it's base64_ approach 👍

Reply:
Done